packages/api/cms-api/src/public-upload/public-uploads.service.ts

```diff
@@ -38,4 +38,24 @@ export class PublicUploadsService {
 
         return publicUpload;
     }
+
+    async getFileById(id: string): Promise<PublicUpload> {
+        const file = await this.publicUploadsRepository.findOne(id);
+        if (!file) {
+            throw new NotFoundException();
```

Comment:
Services shouldn't throw HTTP exceptions, this should be done by the controller

Reply:
imho you don't need this repository wrapper method at all

Reply:
> imho you don't need this repository wrapper method at all

Sorry, I'm a nestjs noob 😅 
Do you mean, I should just do it all in the controller directly instead of partially in the service?